README.md

```diff
@@ -1 +1,16 @@
-# skillforge
\ No newline at end of file
+## Mutli-Platform E-Commerce Test Suite ##
+
+Welcome to our Multi-Platform E-Commerce Test Suite project! This test suite has been developed collaboratively by a team of QA professionals who are passionate about quality. 
+
+# Purpose
+
+The primary purpose of this test suite is to provide a comprehensive set of automated test cases for web, mobile, and API components of e-commerce applications. It aims to validate various functionalities, including user interactions, data integrity, security measures, and more. By utilizing this test suite, development teams can enhance their testing efforts and ensure a seamless and bug-free experience for end-users.
+
+# Meet the Team
+
+This test suite is the result of the combined efforts of:
+
+- **[Hieu Bui](https://www.linkedin.com/in/hbui92/)**: [hbui92](https://github.com/hbui92)
+- **[Alex Hawley](https://www.linkedin.com/in/alexhawley/)**: [alxHawley](https://github.com/alxHawley)
+- **[Ashley Mitchell](https://www.linkedin.com/in/ashley-mitchell-a977061/)**: [AshlMitchell](https://github.com/AshlMitchell)
+- **[Yuri Prater](www.linkedin.com/in/yuri-prater-ii)**: [yprater](https://github.com/yprater)
```

Comment:
You could add https:// to your linkedin @yprater for consistency (noted when I was checking the links from VS code and yours didn't open)

Reply:
Thanks for catching that! Thought I had the protocal in there